### ISSUE TITLE
Emote Scrolling issue

### ISSUE THREAD

**Issue by Creic** [Jiiks/BetterDiscordApp#6]
Discord won't scroll down when messages are being sent. Noticed it mostly happens when an emote has been posted (twitch emote) the window seems to stick to the emote and won't scroll down for messages sent after it. 

Seems to only happen with large emotes, that are taller than a line of text.

Sometimes happens without emotes. In minimal mode, less often, the just a line of text will stop the chat from scrolling.

Have freshly installed Discord to test this.

tl;dr Chat doesn't scroll after an emote is posted.

Edit: Any chance of a keybind to enable/disable minimal mode in future?

**Comment by Phoenix100k** [comment]
Seconding this. If a Twitch emote appears anywhere in-line, the full line will not be displayed, and subsequent messages will not scroll the to the newest message.

**Comment by Jiiks** [comment]
This should now be fixed, simply restarting discord will load the new resources. Hotkeys are definitely possible.

**Comment by Creic** [comment]
Appears to be sorted! Thanks for the quick fix.

**Comment by Lombra** [comment]
Still experiencing this. It's not restricted to messages that contain emotes. Using compact chat mode.

**Comment by sheddup** [comment]
Still experiencing this, only tested with cozy chat mode
